src/app/dataset-view/additional-components/dataset-settings-component/dataset-settings.component.html

```diff
@@ -3,85 +3,48 @@
         <div class="pr-md-4 pr-0">
             <div>
                 <nav>
-                    <ul role="list" class="action-list">
-                        <li [ngClass]="{ 'action-list-item--nav-active': true }" class="action-list-item">
-                            <a routerLinkActive="active" data-view-component="true" class="action-list-content">
+                    <ul role="list" class="action-list d-flex flex-column gap-1">
+                        <li
+                            [class.action-list-item--nav-active]="activeTab === settingsTabsEnum.GENERAL"
+                            class="action-list-item"
+                        >
+                            <a
+                                class="action-list-content align-items-center"
+                                (click)="navigateToSection(settingsTabsEnum.GENERAL)"
+                            >
                                 <span class="text-muted mr-2">
                                     <svg-icon name="account"></svg-icon>
                                 </span>
                                 <span class="action-list-item-label"> General </span>
                             </a>
                         </li>
-                        <mat-divider class="mt-3"></mat-divider>
+                        <mat-divider class="my-1"></mat-divider>
+                        <li
+                            [class.action-list-item--nav-active]="activeTab === settingsTabsEnum.SCHEDULING"
+                            class="action-list-item"
+                            *ngIf="isSchedulingAvailable"
+                        >
+                            <a
+                                class="action-list-content align-items-center"
+                                (click)="navigateToSection(settingsTabsEnum.SCHEDULING)"
+                            >
+                                <span class="text-muted mr-1">
+                                    <svg-icon class="text-muted" name="clock"></svg-icon>
+                                </span>
+                                <span class="action-list-item-label"> Scheduling </span>
+                            </a>
+                        </li>
```

Comment:
Maybe something more declarative with a loop. Very similar code, will likely grow quickly with more tabs.

Reply:
Added config for this